### ISSUE TITLE
[OpenBMC] reventlog formatting needs to be fixed as well as sorting. 

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat-core#3880]
Need to change the formatting of reventlog to match what is in Garrison...

```
mid05tor12cn23: 08/31/2017 04:26:15 System Firmware Progress, Starting operating system boot process, call init 19h (FW Boot Progress)
mid05tor12cn23: 08/31/2017 05:30:50 System Firmware Progress, Baseboard or motherboard initialization (FW Boot Progress)
mid05tor12cn23: 08/31/2017 05:31:12 System Firmware Progress, Memory initialization (FW Boot Progress)
mid05tor12cn23: 08/31/2017 05:32:25 System Firmware Progress, Secondary processor(s) initialization (FW Boot Progress)
mid05tor12cn23: 08/31/2017 05:32:26 System Firmware Progress, Starting operating system boot process, call init 19h (FW Boot Progress)
mid05tor12cn23: 08/31/2017 05:32:54 System Firmware Progress, Baseboard or motherboard initialization (FW Boot Progress)
mid05tor12cn23: 08/31/2017 05:33:00 System Firmware Progress, PCI resource configuration (FW Boot Progress)
mid05tor12cn23: 08/31/2017 05:33:03 System Firmware Progress, Starting operating system boot process, call init 19h (FW Boot Progress)
```

```
mid05tor12cn05: Entry 3: 09/07/2017 02:41:31 org.open_power.Host.Event.Error.Event
mid05tor12cn05: Entry 9: 09/07/2017 02:57:52 org.open_power.Host.Event.Error.Event
mid05tor12cn05: Entry 12: 09/07/2017 04:15:29 org.open_power.Host.Event.Error.Event
mid05tor12cn05: Entry 15: 09/07/2017 04:15:37 org.open_power.Host.Event.Error.Event
mid05tor12cn05: Entry 8: 09/07/2017 02:57:50 org.open_power.Host.Event.Error.Event
mid05tor12cn05: Entry 4: 09/07/2017 02:56:13 org.open_power.Host.Event.Error.Event
mid05tor12cn05: Entry 34: 09/07/2017 15:08:31 org.open_power.Host.Event.Error.Event
mid05tor12cn05: Entry 19: 09/07/2017 04:20:55 org.open_power.Host.Event.Error.Event
mid05tor12cn05: Entry 5: 09/07/2017 02:56:18 org.open_power.Host.Event.Error.Event
```

Essentially move the Entry to after the date/time stamp.  

Then as indicated by @ralphbellofatto  in https://github.ibm.com/DCS-research/R92-cluster/issues/1829 the timestamp ordering is wrong. 

The entries do not give much human readable events, working on this with RAS team.

**Comment by gurevichmark** [comment]
Fixed by PR #3911

**Comment by zet809** [comment]
hi, @whowutwut , shall we close this issue if it is fixed?
